fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/ArithmeticMaker.java

```diff
@@ -66,10 +87,18 @@ public ArithmeticMaker(MetricDictionary metricDictionary, ArithmeticPostAggregat
     }
 
     @Override
-    protected LogicalMetric makeInner(LogicalMetricInfo logicalMetricInfo, List<String> dependentMetrics) {
-        // Get the ArithmeticPostAggregation operands from the dependent metrics
+    public ResultSetMapper makeCalculation(
+            final LogicalMetricInfo logicalMetricInfo, final List<LogicalMetric> dependentMetrics
```

Comment:
Are we intentionally making these parameters final? I didn't think that was a thing we typically did. Also,
the parameters should each be on their own line if we're wrapping.

Reply:
Nope. not intentional.